openeogeotrellis/backend.py

```diff
@@ -2346,12 +2338,30 @@ def as_arg_element(dependency: dict) -> dict:
                 use_pvc=use_pvc,
                 access_token=user.internal_auth_data["access_token"],
             )
+            persistentvolume_batch_job_results_dict_ = k8s_render_manifest_template(
+                "persistentvolume_batch_job_results.yaml.j2",
+                job_name=spark_app_id,
+                job_namespace=pod_namespace,
+                output_dir=output_dir,
+                swift_bucket=bucket,
+            )
 
-            dict_ = yaml.safe_load(rendered)
+            persistentvolumeclaim_batch_job_results_dict_ = k8s_render_manifest_template(
+                "persistentvolumeclaim_batch_job_results.yaml.j2",
+                job_name=spark_app_id,
+            )
 
             with self._double_job_registry as dbl_registry:
                 try:
-                    submit_response = api_instance.create_namespaced_custom_object("sparkoperator.k8s.io", "v1beta2", pod_namespace, "sparkapplications", dict_, pretty=True)
+                    api_instance_core.create_persistent_volume(persistentvolume_batch_job_results_dict_, pretty=True)
+                except ApiException as e:
+                    print("Exception when calling CoreV1Api->create_persistent_volume: %s\n" % e)
```

Comment:
avoid using `print`, that will ruin JSON based logging.

instead use something like this:
```suggestion
                    log.exception(f"CoreV1Api->create_persistent_volume failed: {e}")
```

Moreover, isn't it better to _not_ catch the exception and let the job creation fail as a whole?